### DISCUSSION TITLE
Being hacked

### DISCUSSION THREAD

**Discussion by FuckGlen** [community/community#49354] [New to GitHub] [answered]
I have been getting hacked for 3 months now. I can see where coding has changed and is rerouting and denying access and it all started with my Microsoft account and someone starting a GitHub account on my computer. Can someone please help me fix this chaos?

**Comment by rauzerfabiano** [comment]
If you suspect that your Microsoft account has been compromised, the first step is to immediately change your password to a strong, unique password that you haven't used before. You should also enable two-factor authentication, which adds an extra layer of security to your account.

As for the changes in your coding and rerouting, it's possible that your computer may have been infected with malware or a virus. You can run a full virus scan using reputable antivirus software to check for any malicious programs. You should also check your computer for any unauthorized remote access tools or software that may have been installed.

Regarding the GitHub account, you should immediately change your password and enable two-factor authentication on that account as well. If you suspect that someone has been using your GitHub account to access your code, you should review your account activity and revoke any access tokens that you don't recognize.

If you are unable to resolve the issue on your own, it may be best to seek the help of a professional in the field of cybersecurity or computer forensics.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @FuckGlen 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
